pkg/kudoctl/cmd/install/install.go

```diff
@@ -83,25 +105,29 @@ func verifyFrameworks(args []string) error {
 // Todo: needs testing
 // verifySingleFramework is the umbrella for a single framework installation that gathers the business logic
 // for a cluster and returns an error in case there is a problem
-func verifySingleFramework(name, previous string, gc *github.Client, k2c *kudo.Client) error {
-	// Get most recent ContentDir for selected Framework
-	content, err := gc.GetMostRecentFrameworkContentDir(name)
+func verifySingleFramework(name, previous string, r repo.FrameworkRepository, kc *kudo.Client) error {
+
+	i, err := r.IndexFile.Get(name, vars.PackageVersion)
 	if err != nil {
-		return errors.Wrap(err, "sorting most recent content dir")
+		return errors.Wrapf(err, "getting %s in index file", name)
 	}
 
-	if vars.PackageVersion != "" {
-		content, err = gc.GetSpecificFrameworkContentDir(name)
+	// checking if bundle exists locally already
+	bundleName := i.Name + "-" + i.Version
+	bundlePath := vars.RepoPath + "/" + bundleName
```

Comment:
can these be pulled out into formatString consts (in this package or another) that we then use fmt.Sprintf on?

Reply:
> can these be pulled out into formatString consts (in this package or another) that we then use fmt.Sprintf on?

Mhh 🤔that bundlePath depends on the Framework we are iterating over and would change based on the recursive function when looking at dependencies to be installed, right? Should it be then a consts if it is subject to change and for what else would we need it for?

Reply:
nvm see what you saying and agree :) https://github.com/kudobuilder/kudo/pull/279/commits/4db20b07919ed6257a970934fe3259d25440a1b5